src/nts_ke/server/connection.rs

```diff
@@ -200,6 +234,57 @@ impl KeServerConn {
 
             let keys = gen_key(&self.tls_session).unwrap();
 
+            while self.ntske_state.finished == false {
```

Comment:
Why not `!self.ntske_state.finished`?

Reply:
Oh, I had referenced [nts_ke/client](https://github.com/cloudflare/cfnts/blob/4b7c96b9cb57e9c81b73acfd8fb4d0a1161965c6/src/nts_ke/client.rs#L214). Modified these!